doc/source/how-to-upgrade-to-flower-next.rst

```diff
@@ -189,6 +219,30 @@ Deployment
     # Start a Superlink
     $ flower-superlink --insecure
 
+    # Flower v1.13
+    # In a new terminal window, start a long-running SuperNode
+    $ flower-supernode \
+         --insecure \
+         --superlink 127.0.0.1:9092 \
+         --node-config "partition-id=0 num-partitions=2" \
+         --supernode-address 127.0.0.1:9094 \
+         --isolation subprocess
```

Comment:
```suggestion
         --supernode-address 127.0.0.1:9094
```

`--isolation subprocess` is the default mode, so we can omit this.

Reply:
Fixed.